### DISCUSSION TITLE
Using Generative AI to Improve Image Filenames

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#12] [Comments]
# Using Generative AI to Improve Image Filenames

Can AI help us provide better filenames for images?

https://www.raymondcamden.com/2024/01/26/using-generative-ai-to-improve-image-filenames

**Comment by JonasEriksson** [comment]
That is so cool. Last month a client asked this question - "we have thousands of images named nonsense, surely that's not so great for SEO?" this could solve that :) (obviously needs to updating in the database as well).

**Reply by cfjedimaster** [reply]
Glad you like it. Now - I will say, for thousands of images, you want to check $$ first, but if they're cool with the cost, it's technically a one time operation.
